### PR TITLE
Some FAQ tips on how to improve CleanLearning

### PR DESCRIPTION
**Pull request by jwmueller**

### CODE REVIEW

File: docs/source/tutorials/faq.ipynb [hunk 1]
```diff
@@ -68,16 +68,26 @@
...
   },
   {
    "cell_type": "markdown",
    "id": "d13c9cd0",
    "metadata": {},
    "source": [
-    "At this time, CleanLearning only works directly with numpy matrices or pd.DataFrames.\n",
-    "You can still use cleanlab with other data formats though! Just separately obtain predicted probabilities from your model via cross-validation and pass them as inputs."
+    "At this time, CleanLearning only works with data formatted as numpy matrices or pd.DataFrames, \n",
+    "and with models that are compatible with the `sklearn` API.\n",
```
Comment (cgnorthcutt) [inline docs/source/tutorials/faq.ipynb:80]: Add "Note skorch can be used for PyTorch compatibility. Similarly [KerasClassifier](https://www.tensorflow.org/api_docs/python/tf/keras/wrappers/scikit_learn/KerasClassifier) and [KerasRegressor](https://www.tensorflow.org/api_docs/python/tf/keras/wrappers/scikit_learn) can be used for Tensorflow/Keras compatibility."

File: docs/source/tutorials/faq.ipynb [hunk 2]
```diff
@@ -68,16 +68,26 @@
...
+    "You can still use cleanlab with other data formats though! Just separately obtain predicted probabilities (`pred_probs`) from your model via cross-validation and pass them as inputs.\n",
```
Comment (cgnorthcutt) [inline docs/source/tutorials/faq.ipynb:81]: link 'separately obtain predicted probabilities' to a tutorial where this is done or the the cleanlab.count.estimate_cv_predicted_probabilities here https://github.com/cleanlab/cleanlab/blob/master/cleanlab/count.py#L911

File: docs/source/tutorials/faq.ipynb [hunk 3]
```diff
@@ -68,16 +68,26 @@
...
+    "\n",
+    "\n",
+    "If CleanLearning is running successfully but not improving predictive accuracy of your model, here are some tips:\n",
+    "\n",
+    "1. Use cleanlab to find label issues in your test data as well (we recommend pooling `labels` across both training and test data into one input for `find_label_issues()`). Then manually review and fix label issues identified in the test data to verify accuracy measurements are actually meaningful.\n",
+    "\n",
+    "2. Try different values for `filter_by`, frac_noise`, and `min_examples_per_class` which can be set via the `find_label_issues_kwargs` argument in the initialization of `CleanLearning()`. \n",
+    "\n",
+    "3. Try to find a better model (eg. via hyperparameter tuning or changing to another classifier). `CleanLearning` can find better label issues by leveraging a better model, which allows it to produce better quality training data. This can form a virtuous cycle in which better models -> better issue detection -> better data -> even better models!"
```
Comment (cgnorthcutt) [inline docs/source/tutorials/faq.ipynb:90]: all good. here are some others.

4. Try combining (using AND operator in boolean space) any subset of the filters for more restrictive filtering.
5. Try jointly hyper-parameter tuning both the model and filter_by parameter simultaneously
6. Does your dataset have 'clutter' class or a 'junk' class? If you have bad data, consider creating one (e.g. Caltech-256)
7. Try merging similar/overlapping classes for significantly better performance (e.g. see this tutorial: https://docs.cleanlab.ai/v2.0.0/tutorials/indepth_overview.html#Workflow-3:-Use-cleanlab-to-find-dataset-level-and-class-level-issues)
8. If your `pred_probs` are obtained via a neural network, try averaging the `pred_probs` over the last K epochs of training instead of just using the final `pred_probs`. Similarly, you can try averaging `pred_probs` from several models (remember to re-normalize).

**Review by cgnorthcutt — CHANGES_REQUESTED** [review]

File: docs/source/tutorials/faq.ipynb (hunk 1 above)
Comment (jwmueller) [reply docs/source/tutorials/faq.ipynb:80]: added replacing the latter two with scikeras. I think those two have been removed from public TF API in versions >=2.7:  https://www.tensorflow.org/versions/r2.7/api_docs/python/tf/keras/wrappers/scikit_learn

Comment (jwmueller) [reply docs/source/tutorials/faq.ipynb:80]: Just FYI because of this removal and the fact that scikeras is missing some desirable features, I've actually added our keras wrapper to `cleanlab.experimental` in another PR: 

https://github.com/cleanlab/cleanlab/blob/3f29a5d3a73d04844fff6d3c11b1ab6244ae2670/cleanlab/experimental/keras.py

File: docs/source/tutorials/faq.ipynb (hunk 2 above)
Comment (jwmueller) [reply docs/source/tutorials/faq.ipynb:81]: added

File: docs/source/tutorials/faq.ipynb (hunk 3 above)
Comment (jwmueller) [reply docs/source/tutorials/faq.ipynb:90]: I don't think 4 or 8 are specific to CleanLearning?  Trying to keep this question just about CleanLearning, feel free to add a more general tips section for finding better label issues in separate PR

Comment (jwmueller) [reply docs/source/tutorials/faq.ipynb:90]: added the other points

Comment (cgnorthcutt) [reply docs/source/tutorials/faq.ipynb]: why not just add 4 and 8 at the bottom as its own section "General Tips that might also help improve model accuracy"? Can create a new FAQ later, but adding them there now can't hurt?

Comment (jwmueller) [reply docs/source/tutorials/faq.ipynb]: Sure I'm onboard with that, can you add them?  I won't edit this PR anymore.

**Review by cgnorthcutt — APPROVED** [review]
